app/uploaders/disability_claim_document_uploader.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class DisabilityClaimDocumentUploader < CarrierWave::Uploader::Base
+  MAX_FILE_SIZE = 25.megabytes
+
+  before :store, :validate_file_size
+
+  def initialize(user_uuid, tracked_item_id)
+    super
+    @user_uuid = user_uuid
+    @tracked_item_id = tracked_item_id
+    set_storage_options!
+  end
+
+  def store_dir
+    "disability_claim_documents/#{@user_uuid}/#{@tracked_item_id}"
+  end
+
+  def extension_white_list
+    %w(pdf gif tiff tif jpeg jpg bmp txt)
+  end
+
+  def move_to_cache
+    false
+  end
+
+  private
+
+  def validate_file_size(file)
+    raise CarrierWave::UploadError, 'File size larger than allowed' if file.size > MAX_FILE_SIZE
```

Comment:
This error message does not need to be serialized and shown to the user, correct? We're doing something somewhat similar in message.rb. For us we're not using carrierwave, instead validating  within our model, but it might be a slightly different constraint for you. If however you are showing the error to the user, maybe consider adding it as a validation error.